core/db/migrate/20210713131614_add_unique_index_on_property_id_and_product_id_from_product_properties.rb

```diff
@@ -0,0 +1,7 @@
+class AddUniqueIndexOnPropertyIdAndProductIdToProductProperties < ActiveRecord::Migration[6.1]
+  def change
+    unless index_exists?(:spree_product_properties, [:property_id, :product_id])
+      add_index :spree_product_properties, [:property_id, :product_id], unique: true
```

Comment:
@szymoniwacz before applying that index we need to make sure we don't have any duplicates, eg. https://github.com/spree/spree/blob/master/core/db/migrate/20131211192741_unique_shipping_method_categories.rb